### PR TITLE
Release 1.0.3

### PR DESCRIPTION
**Pull request by jdforrester**
Take the opportunity to clean up and modernise the repo a bit.

### CODE REVIEW

**Comment by santhoshtr** [comment]
Thanks @jdforrester !

File: package.json [hunk 1]
```diff
@@ -38,16 +40,7 @@
...
-    {
-      "type": "GPL",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/GPL-LICENSE"
-    },
-    {
-      "type": "MIT",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/MIT-LICENSE"
-    }
-  ],
+  "license": "(MIT OR GPL-2.0)",
```
Comment (Nikerabbit) [inline package.json]: According to http://spdx-legal.spdx.narkive.com/NGgCHfMa/spdx-2-0-spec-draft-seeking-community-feedback-for-the-license-expression-syntax-section the "OR" is correct.

GPL-2.0 is missing "+".

File: src/jquery.i18n.language.js [hunk 2]
```diff
@@ -5,262 +5,263 @@
 	var language = {
 		// CLDR plural rules generated using
 		// libs/CLDRPluralRuleParser/tools/PluralXML2JSON.html
```
Comment (Nikerabbit) [inline src/jquery.i18n.language.js]: Should the parser be changed to generate JS or should we just exempt this file?

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ jQuery.i18n is a jQuery based Javascript internationalization library. It helps
 
 This is a project by Wikimedia foundation's [Language Engineering team](http://wikimediafoundation.org/wiki/Language_Engineering_team) and used in some of the Wikimedia Foundation projects like Universal Language Selector.
 
-The jquery.i18n library uses a json based localization file format and it is now used as localization file format for  MediaWiki.
+The jquery.i18n library uses a json based localization file format, "banana", which is used as the localization file format for  MediaWiki and other projects.
```
Comment (Nikerabbit) [inline README.md]: The double space after "for" could be removed.

File: README.md [hunk 4]
```diff
@@ -229,12 +229,12 @@ It is also possible to refer messages from an external URL. See below example
 
 ```javascript
 $.i18n().load( {
-	'en': {
-		'message_hello': 'Hello World',
-    	'message_welcome': 'Welcome'
+	en: {
+		message_hello: 'Hello World',
+    	message_welcome: 'Welcome'
```
Comment (Nikerabbit) [inline README.md]: This line seems to be mixing tabs and spaces.

File: README.md [hunk 5]
```diff
@@ -242,15 +242,15 @@ Messages for a locale can be also loaded in parts. Example
...
```
Comment (Nikerabbit) [inline README.md]: This line seems to be mixing tabs and spaces.

File: README.md [hunk 6]
```diff
@@ -242,15 +242,15 @@ Messages for a locale can be also loaded in parts. Example
...
 	}
 } );
 
 $.i18n().load( {
     	// This does not remove the previous messages.
```
Comment (Nikerabbit) [inline README.md]: This line seems to be mixing tabs and spaces.

File: package.json (hunk 1 above)
Comment (jdforrester) [inline package.json]: The library didn't have anything saying "or later" in it, so I took the safer option. Trivial to fix if it was GPL2+ all the time.

File: README.md (hunk 3 above)
Comment (jdforrester) [inline README.md]: Yeah, the README has a few other tweaks that'd be good; I stopped myself from making too many before people had seen it yet. :-)

File: package.json (hunk 1 above)
Comment (Nikerabbit) [inline package.json]: That's true there isn't a central place to define the licenses, other than this file now. The files I spot checked all say "GNU General Public Licence 2.0 or later" and that's also the license we use for other code.

**Pull request merged**